skeleton/config/services.yml.twig

```diff
@@ -19,7 +19,8 @@ services:
         class: {{ EXTENSION.vendor_name }}\{{ EXTENSION.extension_name }}\controller\acp_controller
         arguments:
             - '@config'
-            - '@language'
+{% if LANGUAGE.s_32 %}            - '@language'
+{% endif %}
```

Comment:
This should be able to work as:
```
{% if LANGUAGE.s_32 %}
            - '@language'
{% endif %}
```
If there are for any reason extra whitespace issues, look at controlling that with the `{%-   -%}` forms